mne/beamformer/_dics.py

```diff
@@ -494,97 +499,6 @@ def apply_dics_csd(csd, filters, verbose=None):
             frequencies)
 
 
-def _apply_old_dics(data, info, tmin, forward, noise_csd, data_csd, reg,
```

Comment:
I think we kept that one around because the new one deviates quite a bit from the old one? @wmvanvliet

Reply:
It's in the `git history` so we can always get it back. In the meantime we are taking an unnecessary coverage hit